src/Parallel/Main.hpp

```diff
@@ -180,7 +180,7 @@ Main<Metavariables>::Main(CkArgMsg* msg) noexcept
 
     if (parsed_command_line_options.count("check-options") != 0) {
       // Force all the options to be created.
-      options_.template apply<option_list>([](auto... args) {
+      options_.template apply<option_list, Metavariables>([](auto... args) {
```

Comment:
There are two more calls to `apply` below that also need the parameter.

Reply:
Thank you! :)